### ISSUE TITLE
[FE] Personal information revisions: Update Gender identity and Sexual orientation so they are multiselect

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#36168]
### Background

Related design ticket is #36165

According to Lexi, some of the health systems support multiselect for gender identity and sexual orientation, so we should update our designs to make these fields multiselect.

### Tasks

- [x]  Update Gender identity so it is multiselect
- [x]  Update Sexual orientation so it is multiselect
- [x] For Gender identity, if someone selects "prefer not to answer" after having already checked other boxes, this should uncheck all other options
- [x] For Sexual orientation, if someone selects "prefer not to answer" after having already checked other boxes, this should uncheck all other options
- [ ] Show a specific error message if someone attempts to select other options after selecting "prefer not to answer" - see below
![image](https://user-images.githubusercontent.com/73354907/153667787-b7d3af5e-3702-4da6-83ec-0702bc6d071e.png)

### Acceptance criteria

- [ ] Both gender identity and sexual orientation are multiselect
- [ ] Selecting "prefer not to answer" for both gender identity and sexual orientation will uncheck other selections if they've been made prior to that box being checked
- [ ] Mock up recommendation for showing a message if someone attempts to select other options after selecting "prefer not to answer"

**Comment by adamwhitlock1** [comment]
@MarciMcGuireGCIO I'm pushing one more PR up that should mitigate rendering issues in staging, then ill let you know when it is ready to validate again. The problem occurs when the api doesn't return any data for the new fields, so really this issue won't be present once the vets-api returns the right data for users.

**Comment by MarciMcGuireGCIO** [comment]
Excellent; thank you @adamwhitlock1 !

**Comment by MarciMcGuireGCIO** [comment]
@adamwhitlock1 since this is just waiting on PR, I'm closing, but LMK if any new issues come up and I'll make a new ticket.
